### ISSUE TITLE
Test tolerance a bit low

### ISSUE THREAD

**Issue by KristofferC** [PerezHz/TaylorIntegration.jl#79]
On the upcoming 1.3 this package errors with

```
Complex dependent variable: Test Failed at /root/.julia/packages/TaylorIntegration/R6yrv/test/taylorize.jl:198
  Expression: norm(abs2(xv1p[end]) - abs2(exact_sol(tv1p[end], cc, cx0)), Inf) < 1.0e-15
   Evaluated: 2.1094237467877974e-15 < 1.0e-15
Stacktrace:
 [1] top-level scope at /root/.julia/packages/TaylorIntegration/R6yrv/test/taylorize.jl:198
 [2] top-level scope at /workspace/srcdir/julia/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /root/.julia/packages/TaylorIntegration/R6yrv/test/taylorize.jl:153
Test Summary:              | Pass  Fail  Total
Complex dependent variable |   17     1     18
```

It seems that the test tolerance is put a bit too tightly?

**Comment by lbenet** [comment]
Thanks for reporting! The problem seems already present in Julia 1.3-rc1.0. 

The specific tests are related to some way of optimizing the integration for dependent complex variables (using `@taylorize`), which creates a new (internal) function that should do exactly the same for the integration as the "default" one, just faster. This seems to be ok. Yet, the broken test checks the absolute accuracy of the numerical integration vs the exact solution, and the numerical solution is what has changed. 

@KristofferC Has something (a library or some Julia implementation) related with additions or multiplications of complex numbers changed in Julia 1.3-rc1 in comparison to Julia 1.2? Icouldn't find anything like that in the NEWS.md file...

**Comment by KristofferC** [comment]
Not sure. One way to debug it might be to use Debugger.jl and step through the functions in both Julia versions and see where the difference occurs.

**Comment by lbenet** [comment]
Fixed in #80
